sdm.assignment9.Core.Entity/Class1.cs

```diff
@@ -4,5 +4,11 @@ namespace sdm.assignment9.Core.Entity
 {
     public class Class1
     {
+
+        public void DoTheThing()
```

Comment:
Awful method name.
No comments.

Reply:
The method name is valid because it describes what it does. It does something which is important to understand

Reply:
The method name is to vague, this should be changed to describe what it does more specific

Reply:
I am leaving this group if you don't accept this build